composer.json

```diff
@@ -2,7 +2,7 @@
   "name": "imgix/imgix-php",
   "description": "A PHP client library for generating URLs with imgix.",
   "type": "library",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
Could you back this change out? It's generally cleaner for us to version-incrementing after the change in question has hit master (and also I think this qualifies for a minor-version bump, since we're adding to the interface).